src/pip/_internal/commands/debug.py

```diff
@@ -7,9 +7,13 @@
 import logging
 import os
 import sys
+from types import ModuleType
```

Comment:
This line can be moved inside the `MYPY_CHECK_RUNNING` block (and you can order them alphabetically ;) )

Reply:
Done, thanks a lot for your awesome feedback :D